Find404.scala

```diff
@@ -0,0 +1,61 @@
+import java.io._
+import scala.io.Source._
+import scala.io.StdIn._
+import scala.collection.mutable.ArrayBuffer
+
+var fileSet = ArrayBuffer[File]()
+var ignoreSet = ArrayBuffer[String]()
+val directorySet = new File(args(0))
```

Comment:
What happens when a user doesn't pass in any `args`. Generally good command line apps will give a message explaining roughly how to use this application. I think for now a sentence or two would suffice. And then you exit.